File: contests-app.API/Models/Case.cs
namespace contests_app.API.Models
{
    public class Case
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
        public User Owner { get; set; }
    }
}

File: contests-app.API/VueClientApp/src/models/Case.ts
import { User } from '@/models/User.ts'

export class Case {
  constructor(caseItem?: Partial<Case>) {
    this.id = caseItem?.id ?? ''
    this.title = caseItem?.title ?? ''
    this.description = caseItem?.description ?? ''
    this.imageUrl = caseItem?.imageUrl ?? ''
    this.owner = caseItem?.owner ? new User(caseItem.owner) : undefined
  }

  id: string
  title: string
  description: string
  imageUrl: string
  owner: User | undefined
}
